code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -944,3 +944,38 @@
 	implants = list(/obj/item/organ/cyberimp/eyes/hud/security)
 
 	back = /obj/item/storage/backpack
+
+
+
+//Library spawn
+/obj/effect/mob_spawn/human/library
+	death = FALSE
+	roundstart = FALSE
+	random = FALSE
+	name = "records librarian sleeper"
+	desc = "A humming sleeper with a silhouetted occupant inside. It's just a front, stored underground are many librarians."
+	mob_name = "Records Librarian"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	outfit = /datum/outfit/librarian
+	short_desc = "A place for players to read up on lore and information while not affecting the actual round \
+	When you are done, Cryo and respawn to join the game."
+	assignedrole = "Lobotomy Corporation Records Librarian"
+
+/obj/effect/mob_spawn/human/library/Destroy()
```

Comment:
Now that I think about it, why are you doing this all? Ghost role spawners have variables for infinite "spawn slots" already.

Reply:
I didn't see it when I was working on the intern, so I just winged it. This works fine but I will fix